core/pom.xml

```diff
@@ -390,10 +390,19 @@
     </dependency>
 
     <!-- Print service: MapFish -->
+    <dependency>
+      <groupId>com.github.librepdf</groupId>
+      <artifactId>openpdf</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.mapfish.print</groupId>
       <artifactId>print-lib</artifactId>
       <exclusions>
+        <exclusion>
+          <!-- provided by openpdf -->
+          <groupId>com.itextpdf</groupId>
+          <artifactId>itextpdf</artifactId>
+        </exclusion>
```

Comment:
This exclusion can probably be removed now that we are using mapfish-print 2.2.0 (that no longer depends on itextpdf.